utils/SubHeading/index.js

```diff
@@ -0,0 +1,10 @@
+import { StyledDiv, StyledH2 } from "./SubHeading.style";
+
+export default function SubHeading({ children }) {
+  return (
+    <StyledDiv>
+      <StyledH2>{children}</StyledH2>
+      <hr></hr>
+    </StyledDiv>
+  );
+}
```

Comment:
As I mentioned before I would like to ask you to invest some time in naming your components. 
Usually we want to avoid using <hr/> elements, but would like to solve the representation via CSS.  

```suggestion
export default function SubHeading({ children }) {
  return (
    <StyledDiv>
      <StyledH2>{children}</StyledH2>
      <hr />
    </StyledDiv>
  );
}
```

Reply:
solved it with CSS using border-top